.vscode/scripts/feeder_config/dogmode/DogMode.dbc

```diff
@@ -0,0 +1,98 @@
+VERSION ""
+
+
+NS_ :
```

Comment:
I know that you want to have this to be in sync, but I still think that is confusing since we don't have the dogmode as app